sdk/keyvault/azkeys/client.go

```diff
@@ -284,15 +287,36 @@ type CreateRSAKeyOptions struct {
 
 	// Application specific metadata in the form of key-value pairs.
 	Tags map[string]string `json:"tags,omitempty"`
+
+	// Elliptic curve name. For valid values, see JsonWebKeyCurveName.
+	Curve *JSONWebKeyCurveName `json:"crv,omitempty"`
+
+	// The attributes of a key managed by the key vault service.
+	KeyAttributes *KeyAttributes         `json:"attributes,omitempty"`
+	KeyOps        []*JSONWebKeyOperation `json:"key_ops,omitempty"`
+
+	// The policy rules under which the key can be exported.
+	ReleasePolicy *KeyReleasePolicy `json:"release_policy,omitempty"`
 }
 
 // convert CreateRSAKeyOptions to generated.KeyCreateParameters
 func (c CreateRSAKeyOptions) toKeyCreateParameters(k KeyType) generated.KeyCreateParameters {
+	var keyOps []*generated.JSONWebKeyOperation
+	if c.KeyOps != nil {
+		keyOps = make([]*generated.JSONWebKeyOperation, len(c.KeyOps))
+		for i, k := range c.KeyOps {
+			keyOps[i] = (*generated.JSONWebKeyOperation)(k)
+		}
+	}
 	return generated.KeyCreateParameters{
```

Comment:
Given the method is private, can I assume `generated.KeyCreateParameters` is not exposed? We generally follow `<MethodName>Options` for public inputs.

Reply:
Your assumption is correct, anything in the `generated` subpackage is not exposed. This method has a `CreateRSAKeyOptions` struct.